### DISCUSSION TITLE
mongo Query Error: Cannot convert undefined or null to object

### DISCUSSION THREAD

**Discussion by Valdri** [Budibase/budibase#6316] [Help] [answered]
I'm trying to update status for specific product  _id, but I get "Cannot convert undefined or null to object" error 
![Zrzut ekranu z 2022-06-14 18-32-52](https://user-images.githubusercontent.com/13934649/173629675-79120218-a963-4297-b5b9-7cee7bda29d8.png)

**Comment by melohagan** [comment]
Hi @Valdri 

Is it an object id? Then it would be `"_id": "ObjectId('{{id}}')"`

**Comment by Valdri** [comment, marked as answer]
Hi @melohagan 
still not working
I also tried adding all the values, but that doesn't help either.
the budibase query I tried:
`{
	"_id": "ObjectId('{{id}}')",
  "products_id": "419539186742564520",
  "name": "Kissenbezug für das Modell Beddinge - Braun L03",
  "order_id": "419539186",
  "quantity": "2",
  "sku": "B03",
  "status": "test"
}`
Here is how this data looks like in mongo express:
`{
    _id: ObjectId('62a8a9ceea896c6a34deb97e'),
    products_id: '419539186742564520',
    name: 'Kissenbezug für das Modell Beddinge - Braun L03',
    order_id: 419539186,
    quantity: 2,
    sku: 'B03',
    status: 'default'
}`

**Reply by melohagan** [reply]
Sorry I just realised this is an **updateOne**. You need two comma separated objects (MongoDB is next on my list of docs to update)

https://www.mongodb.com/docs/drivers/node/current/usage-examples/updateOne/

Something like this:
```
{
  "_id": "ObjectId('update_the_status_of_this_doc')"
},
{
  "$set": { "status": "NEW_STATUS" }
}
```

**Reply by Valdri** [reply]
it works
Thanks for the help

**Reply by melohagan** [reply]
You're welcome!
